### PR TITLE
[RFR] Add tag move add edit

### LINKED ISSUE #28: Add the ability to move a tag from the cli

### PR DESCRIPTION
**Pull request by floo51**
Fixes #28

### CODE REVIEW

File: api/src/domain/configurations/version.js [hunk 1]
```diff
@@ -10,7 +10,7 @@ export const get = async (projectId, environmentName, configName, tagName) => {
     if (!tag) {
         version = await versionsQueries.findOneByHash(configuration.id, tagName);
     } else {
-        version = await versionsQueries.findOneByTag(configuration.id, tag.id);
+        version = await versionsQueries.findOne({ id: tag.version_id });
```
Comment (Kmaschta) [inline api/src/domain/configurations/version.js]: Finally, it works like this. So I don't touch it.

File: api/src/queries/versions.js [hunk 2]
```diff
@@ -46,7 +51,7 @@ const find = async (configurationId) => {
 const findOneByHash = async (configurationId, hash) => {
```
Comment (Kmaschta) [inline api/src/queries/versions.js]: This query didn't works because of the missing `hash` field in `searchableFields`.

**Comment by Kmaschta** [comment]
RFR. A twofold review is hardly a luxury for this one.

File: api/src/domain/tags/add.js [hunk 3]
```diff
@@ -0,0 +1,19 @@
+import { get as getVersion } from '../configurations/version';
+import tagsQueries from '../../queries/tags';
+import validateTag from './validator';
+
+export default async (projectId, environmentName, configName, selector, rawName) => {
+    const name = rawName.toLowerCase();
```
Comment (floo51) [inline api/src/domain/tags/add.js]: Can you check this in `validateTag` and throw an error if it's not all lower case ?

File: cli/src/commands/tag.js [hunk 4]
```diff
@@ -0,0 +1,112 @@
...
+
+    const environment = getOption('e', 'environment');
+    const configuration = getOption('c', 'configuration');
+    const tag = getOption('t', 'tag');
+    const hash = getOption('hash', 'hash');
+
+    const project = yield modules.project.retrieveFromConfig();
+    const movedTag = yield modules.tag.move(project, environment, configuration, tag, hash);
+
+    ui.print(`${bold('Cool!')} Your new tag "${bold(movedTag.name)}" was successfuly moved to the new configuration.`);
```
Comment (floo51) [inline cli/src/commands/tag.js]: s/successfuly/successfully/

**Pull request merged**